packages/docs/docs/careplans/loinc.md

```diff
@@ -0,0 +1,373 @@
+            "display": "Several days",
+            "system": "http://loinc.org"
+            // highlight-end
+          }
+        },
+        {
+          "valueCoding": {
+            // highlight-start
+            "code": "LA6570-1",
+            "display": "More than half the days",
+            "system": "http://loinc.org"
+            // highlight-end
+          }
+        },
+        {
+          "valueCoding": {
+            // highlight-start
+            "code": "LA6571-9",
+            "display": "Nearly every day",
+            "system": "http://loinc.org"
+            // highlight-end
+          }
+        }
+      ]
+    },
+    {
+      "type": "choice",
+      "code": [
+        {
+          "code": "44255-8",
+          "display": "Feeling down, depressed, or hopeless",
+          "system": "http://loinc.org"
+        }
+      ],
+      "required": false,
+      "linkId": "/44255-8",
+      "text": "Feeling down, depressed, or hopeless",
+      "answerOption": [
+        {
+          "valueCoding": {
+            // highlight-start
+            "code": "LA6568-5",
+            "display": "Not at all",
+            "system": "http://loinc.org"
+            // highlight-end
+          }
+        },
+        {
+          "valueCoding": {
+            // highlight-start
+            "code": "LA6569-3",
+            "display": "Several days",
+            "system": "http://loinc.org"
+            // highlight-end
+          }
+        },
+        {
+          "valueCoding": {
+            // highlight-start
+            "code": "LA6570-1",
+            "display": "More than half the days",
+            "system": "http://loinc.org"
+            // highlight-end
+          }
+        },
+        {
+          "valueCoding": {
+            // highlight-start
+            "code": "LA6571-9",
+            "display": "Nearly every day",
+            "system": "http://loinc.org"
+            // highlight-end
+          }
+        }
+      ]
+    },
+    {
+      "type": "decimal",
+      "code": [
+        {
+          "code": "70272-0",
+          "display": "Patient health questionnaire 4 item total score",
+          "system": "http://loinc.org"
+        }
+      ],
+      "required": false,
+      "linkId": "/70272-0",
+      "text": "Patient health questionnaire 4 item total score",
+      "item": [
+        {
+          "text": "The PHQ-4 is different -- although the total score (which ranges from 0-12) can be used, it is really a combination of the PHQ-2 depression scale (described above) and the GAD-2 anxiety scale (from the parent GAD-7 anxiety scale).  Thus, another way to look at it is a 0-6 depression subscale and a 0-6 anxiety scale. It is clearly different than either the PHQ-9 or the PHQ-2",
+          "type": "display",
+          "linkId": "/70272-0-help"
+        }
+      ]
+    }
+  ]
+}
+```
+
+</details>
+
+## See Also
+
+- <sup id="ref1">1</sup> https://loinc.org/get-started/scope-of-loinc/
```

Comment:
The `<sup>` looks a little weird.  Maybe add square brackets?  `<sup>[1]</sup>`

The links are not rendering as links.  I think you can fix by surrounding with angle brackets `<https://...>`

<img width="527" alt="image" src="https://github.com/medplum/medplum/assets/749094/4525b631-4fe4-49ca-a6db-2c72b6293680">